djl-spring-boot-starter-parent/pom.xml

```diff
@@ -34,13 +34,13 @@
     </distributionManagement>
 
     <properties>
-        <djl.version>0.6.0</djl.version>
+        <djl.version>0.7.0</djl.version>
         <java.version>11</java.version>
         <jna.version>5.3.0</jna.version>
         <maven-javadoc-plugin.version>3.1.1</maven-javadoc-plugin.version>
         <maven-source-plugin.version>3.1.0</maven-source-plugin.version>
         <mxnet.native.version>1.7.0-a</mxnet.native.version>
```

Comment:
We can remove mxnet.native and pytorch.native version. Both of them are covered by bom now.

Reply:
Do we have a solution when a dependency is needed in a restricted environment (e.g. no internet access)? A lot of customers run EKS for example in private subnets. If no native dependency then an example on how to accomplish the same without it.

Reply:
User can still include native package. with bom, we just don't need specify version any more. The bom will provide correct version

Reply:
Exactly, you still have the same packages it's just that the versions are maintained in one place.

However we cannot change it at the moment as the bom misses the versions for dependencies with classifiers. Removing currently results in an error:
`'dependencies.dependency.version' for ai.djl.mxnet:mxnet-native-mkl:jar:linux-x86_64 is missing. @ line 38, column 17`

Reply:
Oh, I must have misunderstood. If the BOM will provide the right dependency it is actually a lot better :) Before there was a possibility of out of sync which I ran into a few times. Let's remove the versions to avoid confusion.